retry.go

```diff
@@ -247,9 +258,10 @@ func (r *retryHTTP) execute() (res *http.Response, err error) {
 				// This is currently used for Snowflake login. The caller must generate an error object based on HTTP status.
 				break
 			}
+			sampledResponseBody = getBodyTrace(res)
```

Comment:
Is there a specific reason we only want to pull partial error response bodies?

Reply:
They are biggish, mostly html and not that rare.

Reply:
Ok. 5k still seems large if this isn't rare... but it's easy enough to turn off.

Reply:
Because 
1. I'm dumping unconditionally
2. I'm nervous about dumping _huge_ response bodies in the logs